### PR TITLE
Logic and ui for restaurant website with responsive tab navigations

### PR DESCRIPTION
**Pull request by dannisonarias**
create a webpack environment with all configurations and directories
create a barebones HTML file
create modules for each tab
create tab-switching logic inside of index.js

### CODE REVIEW

**Comment by dannisonarias** [comment]
This linter is not working properly with es6 and scss . It doesn't even allow me to use self invoked functions.  TSE please review the project as it is , and if you have issues with the linter , please help me fix the issues don't just point them out.  i know they are there .

File: .stickler.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+# add the linters you want stickler to use for this project
+linters:
+  eslint:
+    
+    # indicate where is the config file for stylelint
+    config: './.eslintrc.json'
+    
+  sasslint:
+    # indicate where is the config file for stylelint
+    config: ./sass-lint.yml
```
Comment (mcrd25) [inline .stickler.yml]: Concerning your sass lint issues:
This file does not exist. it should be your `.stylelintrc.json` not so?
Maybe this should help. Once your config is right it should be fine.

Comment (mcrd25) [inline .stickler.yml]: Also, You did not even have to style lint. but since you did it's best to fix errors.

Comment (mcrd25) [reply .stickler.yml]: If fixing the config here does not help I'd try my best to help troubleshoot with you. 🙂

File: README.md [hunk 2]
```diff
@@ -1,2 +1,41 @@
...
+
+- Github: [@dannisonarias](https://github.com/dannisonarias)
+- Twitter: [@AriasDannison](https://twitter.com/AriasDannison)
+- Linkedin: [Dannison Arias](https://www.linkedin.com/in/dannison-arias-777919190/)
+
+## 🤝 Contributing
+
+Contributions, issues and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/dannisonarias/Javascript_Tic_Tac_Toe/issues).
```
Comment (mcrd25) [inline README.md]: This is linking to another project's issues page. Please fix.

File: README.md [hunk 3]
```diff
@@ -1,2 +1,41 @@
...
+# Screenshot
+![sample](capture.png)
+
+## Built With
+- HTML & SCSS
+- Bootstrap
+- JavaScript/ES6
+- Webpack
+
+## Usage
```
Comment (mcrd25) [inline README.md]: [OPTIONAL]
Consider adding how to actually run the app.

File: README.md [hunk 4]
```diff
@@ -1,2 +1,41 @@
...
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Microverse
+- The Odin Project
+
+## 📝 License
+
+This project is [MIT](./license.md) licensed.
```
Comment (mcrd25) [inline README.md:37]: Good README. 😄

File: package.json [hunk 5]
```diff
@@ -0,0 +1,36 @@
+{
+  "name": "Restaurant-App",
+  "version": "1.0.0",
+  "description": "Restaurant App built with Javascript, using the Webpack library.",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
```
Comment (mcrd25) [inline package.json]: Your README references `build` script but you don't actually have build script which uses your webpack commands

Comment (mcrd25) [reply package.json]: Scripts like `build`, `start`, and `development` are required according to this [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/restaurant_page/scripts_requirement.png) from pathright.

**Review by mcrd25 — CHANGES_REQUESTED** [review]
# Changes Required 🔍 

Hi Team! 

Great job with this project! It's a cool looking cafe! 😄 

***You only have to address some minor required changes (see review comments)***

All you have to do now I guess is address the minor issues and we should be fine!
Let's get these minor issues sorted! 😄 👍 

***Please makes changes and submit another review request.
If you have any questions, let me know here or on slack and I will be glad to clarify with you. You can also reach me and connect with me on my social media or email.***

*Happy coding 😄,*
**[--Maya](https://github.com/mcrd25)**
[GitHub](https://github.com/mcrd25) [Twitter](https://twitter.com/mcrd25) [linkedin](https://www.linkedin.com/in/mayadouglas/) [GitLab](https://gitlab.com/mcrd25)

---
*P.S. all requested changes not marked by [OPTIONAL] or are not positive comments are **REQUIRED***

**Review by bellom — APPROVED** [review]
## Hi @dannisonarias  :clap: 

**Great work making all the necessary corrections from the previous review.**

**NOTE:** Make sure to always set up your linter's config and you should have your linter's check on your `GitHub` page.
checkout this [linters-config#troubleshooting](https://github.com/microverseinc/linters-config#troubleshooting) for help. Example below;
image
If you are not having this check on your `Github` page, kindly visit :point_right:[ linter set up](https://github.com/microverseinc/linters-config/blob/master/ruby/.stickler.yml)

## Congratulation :+1:  
_Go ahead and merge to dev and don't forget to submit project completion form.
Good luck in the next project :slightly_smiling_face:_ 

**Best,**

**Oluwaseun Bello**
**_Slack Handle - @sean bellom_**

**Pull request merged**